### PR TITLE
Add Strato

### PR DESCRIPTION
**Pull request by 40504**

### CODE REVIEW

**Review by Carlgo11 — APPROVED** [review]

**Comment by phallobst** [comment]
This seems to apply only to "HiDrive" (i.e. its cloud storage product). Does it work for the domain account as well? Otherwise, a separate entry in the "Backup" category would probably be better.

File: _data/backup.yml [hunk 1]
```diff
@@ -157,6 +157,15 @@ websites:
       - totp
     doc: https://www.qnap.com/en-us/how-to/tutorial/article/how-to-enhance-account-security-using-2-step-verification
 
+  - name: STRATO
```
Comment (ApCoder123) [inline _data/backup.yml]:
```suggestion
  - name: STRATO HiDrive
```

File: _data/backup.yml [hunk 2]
```diff
@@ -157,6 +157,15 @@ websites:
...
+    url: https://www.strato.de/
+    img: strato.svg
+    tfa:
+      - totp
+    doc: https://www.strato.de/faq/cloud-speicher/2-Faktor-Authentifizierung/
+    regions:
+      - de
```
Comment (ApCoder123) [inline _data/backup.yml:167]:
```suggestion
      - de
      - nl
```
Also is available for Netherlands: https://www.strato.nl/faq/cloud-storage/Twee-factor-authenticatie/

File: img/backup/strato.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg version="1.1" viewBox="0 0 90 90" xmlns="http://www.w3.org/2000/svg"><path d="m57.855 25.713 0.23074 0.0040598c3.446 0.12145 6.1998 2.9526 6.1998 6.4264v51.428l-0.0040571 0.23061c-0.12137 3.4441-2.9507 6.1979-6.4265 6.1979h-51.427l-0.23061-0.004057c-3.4442-0.12137-6.198-2.9506-6.198-6.4244v-51.428l0.0040571-0.23061c0.12137-3.4442 2.9507-6.1998 6.4245-6.1998h51.427zm19.288-25.713c7.1001 0 12.857 5.755 12.857 12.857l-0.00158 44.999c-7.1001 0-12.857-5.755-12.857-12.855l0.00158-25.716c0-3.551-2.8795-6.4285-6.4286-6.4285h-25.714c-7.1001 0-12.857-5.757-12.857-12.857h45z" fill="#FF8802"/></svg>
```
Comment (ApCoder123) [inline img/backup/strato.svg:1]:
```suggestion
<svg viewBox="0 0 90 90" xmlns="http://www.w3.org/2000/svg"><path d="m57.855 25.713 0.23074 0.0040598c3.446 0.12145 6.1998 2.9526 6.1998 6.4264v51.428l-0.0040571 0.23061c-0.12137 3.4441-2.9507 6.1979-6.4265 6.1979h-51.427l-0.23061-0.004057c-3.4442-0.12137-6.198-2.9506-6.198-6.4244v-51.428l0.0040571-0.23061c0.12137-3.4442 2.9507-6.1998 6.4245-6.1998h51.427zm19.288-25.713c7.1001 0 12.857 5.755 12.857 12.857l-0.00158 44.999c-7.1001 0-12.857-5.755-12.857-12.855l0.00158-25.716c0-3.551-2.8795-6.4285-6.4286-6.4285h-25.714c-7.1001 0-12.857-5.757-12.857-12.857h45z" fill="#FF8802"/></svg>
```
The `version` attribute is unnecessary.

**Review by ApCoder123 — CHANGES_REQUESTED** [review]

**Comment by phallobst** [comment]
Sorry, but the 2FA only applies to the HiDrive service. As long as the regular domain/hosting service of strato.de does not provide 2FA as well, we cannot merge this change in the current form. You are welcome to re-open this PR once the situation changes.

**Pull request closed without merging**